### DISCUSSION TITLE
lxqt: incorrect font & cursor size / screen DPI (whitelisting issue)

### DISCUSSION THREAD

**Discussion by hotcapy** [netblue30/firejail#5754] [Q&A] [answered]
Hello!

Whenever I run **KeePassXC** with **Firejail** using default profile, font & mouse cursor size are correct:

![right](https://user-images.githubusercontent.com/80588349/227734354-aa22ca6f-27cd-4814-aa64-3340c8ff700c.png)

But as soon as I add suggested (slightly adopted) **whitelist options** to my `keepassxc.local` file, font in app looks larger then it should, while cursor smaller:

![wrong](https://user-images.githubusercontent.com/80588349/227734292-d5b69505-9597-4b9d-928a-b987688ca3f6.png)

```
env LD_PRELOAD=/usr/lib/libhardened_malloc.so
net none
mkdir ${HOME}/.local/share/passwords
whitelist ${HOME}/.local/share/passwords
mkdir ${HOME}/.local/share/keys
whitelist ${HOME}/.local/share/keys
mkdir ${HOME}/.cache/keepassxc
mkdir ${HOME}/.config/keepassxc
whitelist ${HOME}/.cache/keepassxc
whitelist ${HOME}/.config/keepassxc
include whitelist-common.inc
```

I use **ArchLinux** OS, **LXQt** DE and **Openbox** WM.
**firejail-git 0.9.73** from **AUR**, builded yesterday.
My **fonts** are in `/usr/share/fonts` & **cursor**/icons in `/usr/share/icons`.

Most likely problem is not KeePassXC specific, since I have same cursor look in several other apps using Firejail, for example telegram-desktop with default profile (which seems to use whitelist as well).

Can I fix it someway? Thanks for any possible help!

**Comment by ghost** [comment, marked as answer]
Not really familiar with LXQt, but have you tried whitelisting its configuration path yet (if that is where font/theme settings are stored)? Adding `whitelist ${HOME}/.config/lxqt` should be enough to check whether that has any effect.

Off-topic: `net none` is already in /etc/firejail/keepassxc.profile so there's no need to have it in your local override too.

---

Edit by @kmk3: Fixed in #5755.

**Reply by hotcapy** [reply]
Thank you so much! It worked: font size is normal again :-)

Cursor is still smaller then set in settings, despite it's size is mentioned in `${HOME}/.config/lxqt/session.conf`, but it's ok, nothing so critical, I can live with that. I remember long time ago I've seen same behavior also with other sandboxes and desktops, like Snaps on KDE.

Thanks again for your answer and for tip about `net none`, it was really helpful!
